File: find-triangular-sum-of-an-array.js
const triangularSum = function(nums) {
  while (nums.length > 1) {
    let a = []
    for (let i = 0; i < nums.length - 1; i++) {
      a.push((nums[i] + nums[i+1]) % 10)
    }
    nums = [...a]
  }

  return nums[0]
}

console.log(triangularSum([1,2,3,4,5]))

File: go-archive/find-triangular-sum-of-an-array.go
func triangularSum(nums []int) int {
	for len(nums) > 1 {
		arr := []int{}

		for i := 0; i < len(nums)-1; i++ {
			arr = append(arr, (nums[i]+nums[i+1])%10)
		}
		nums = arr
	}

	return nums[0]
}
